kano_settings/system/display.py

```diff
@@ -185,7 +185,7 @@ def set_hdmi_mode(group=None, mode=None):
     group = group.lower()
     mode = int(mode)
 
-    if group == 'cea':
+    if group == 'cea' or group == '1':
```

Comment:
Can we also include the non-string `1`? It seems that it is likely that you would want to actually call this with integers as this is how you presumed it was behaving.

In which case, we might want to clean up the line with something like

```python
if group in ['cea', '1', 1]:
```

Also, do we trust that `mode` will always be an integer or string representation of an integer, otherwise `mode = int(mode)` above will fail.

Finally, can you add a comment to this function - it may not be a great function but at least having some explanation about what it actually does rather than what we assume it does would be helpful.